pages/table/sticky-header-with-actions.page.tsx

```diff
@@ -22,8 +22,11 @@ export default function () {
     <>
       <h1>Table with a sticky header and actions</h1>
       <ScreenshotArea>
-        <div style={{ height: '400px', width: '500px', overflow: 'auto', padding: '0px 1px' }} id="scroll-container">
-          <div style={{ height: '100px' }} />
+        <div
+          style={{ blockSize: '400px', inlineSize: '500px', overflow: 'auto', padding: '0px 1px' }}
```

Comment:
Should we update the `padding` here too?

Reply:
Good catch! Updating...